reaktive/src/commonTest/kotlin/com/badoo/reaktive/observable/RepeatTest.kt

```diff
@@ -44,7 +44,7 @@ class RepeatTest : ObservableToObservableTests by ObservableToObservableTestsImp
     @Test
     fun emits_all_values_of_first_iteration_WHEN_count_is_negative() {
         val upstream = TestObservable<Int?>()
-        val observer = upstream.repeat(count = -1).test()
+        val observer = upstream.repeat(times = -1).test()
```

Comment:
This test is now invalid, because negative values are now incorrect. It should be `Long.MAX_VALUE`. Same for other places where `-1` is passed.

Reply:
Fixed